dev-proxy/Logging/ProxyConsoleFormatter.cs

```diff
@@ -258,6 +261,7 @@ private static (ConsoleColor bg, ConsoleColor fg) GetMessageTypeColor(MessageTyp
             MessageType.Mocked => (ConsoleColor.DarkMagenta, fgColor),
             MessageType.Failed => (ConsoleColor.DarkRed, fgColor),
             MessageType.Tip => (ConsoleColor.DarkBlue, fgColor),
+            MessageType.Timestamp => (ConsoleColor.Gray, fgColor),

```

Comment:
Let's align it with debug, which is `LogLevel.Debug => (bgColor, ConsoleColor.Gray),`, so that it doesn't stand out as much since it's not of primary concern

Reply:
![image](https://github.com/user-attachments/assets/2023b74a-ff93-44c6-8898-262045f27bec)